liblumen_eir_interpreter/src/native/maps.rs

```diff
@@ -8,6 +8,11 @@ use crate::module::NativeModule;
 pub fn make_maps() -> NativeModule {
     let mut native = NativeModule::new(Atom::try_from_str("maps").unwrap());
 
+    native.add_simple(Atom::try_from_str("find").unwrap(), 2, |proc, args| {
```

Comment:
@KronicDeth Should we generally be exposing BIFs like this in the eir_interpreter project? Most of the BIFs don't show up here, so I wasn't sure whether it was necessary or not

Reply:
It's a nice to have, but not required.  We had been adding the BIFs to the interpreter only as needed, so it was what was needed for spawn-chain and @bcardarella's dbmon (which is in a separate repo).

You'll notice that most of the BIFs are bare native functions in `erlang.rs` and haven't been converted to the mod form with a `code`, `native`, `place_frame_with_arguments`, etc.  This is because they weren't being called in spawn-chain yet.

After I get back from vacation, I'm going to make a proc macro to generate all the code in those modules from just the `native`'s signature and some attributes to set the function name for things like `add_2` being `+` in Erlang.

It is not required that community contributors adding BIFs also add them to the interpreter unless other parts of the community come to expect all BIFs to be in the interpreter and then we'll change the policy.

Reply:
That makes sense, thank you!